data/mods/gen8/random-teams.ts

```diff
@@ -1691,6 +1693,8 @@ export class RandomGen8Teams {
 		teamDetails: RandomTeamsTypes.TeamDetails,
 		species: Species,
 		isDoubles: boolean,
+		preferredType = '',
```

Comment:
Why have these default to `''` when the type after them is a required type? You're always going to be specifying them anyways, so it seems like it'd be better to just make the type explicit instead?

Reply:
Agreed, fixed